src/wallet/wallet2.cpp

```diff
@@ -467,7 +469,7 @@ boost::optional<password_container> wallet2::password_prompt(const bool new_pass
 std::unique_ptr<wallet2> wallet2::make_from_json(const boost::program_options::variables_map& vm, const std::string& json_file)
 {
   const options opts{};
```

Comment:
This object could be created in `generate_from_json` instead to save a parameter argument.